library/Vanilla/Model.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GPLv2
+ */
+
+namespace Vanilla;
```

Comment:
Could we move this to a `Vanilla\DB` or `Vanilla\Database` namespace? I don't want to overcrowd the top level here.

Reply:
I moved this into `Vanilla\Models`. It was an existing namespace and seemed to fit.